docs/SUMMARY.md

```diff
@@ -4,66 +4,74 @@
 
 ## Getting Started
 
-* [Basics of FHE programs](getting-started/fhe_basics.md)
 * [Installation](getting-started/installing.md)
 * [Quick Start](getting-started/quick\_start.md)
 * [Compatibility](getting-started/compatibility.md)
-* [Exactness](getting-started/exactness.md)
-* [Performance](getting-started/performance.md)
-* [Terminology and Structure](getting-started/terminology_and_structure.md)
+* [Terminology](getting-started/terminology.md)
 
-## Tutorials
+## Feature
+
+* [Overview](getting-started/fhe_basics.md)
 
-* [Decorator](tutorial/decorator.md)
-* [Progressbar](tutorial/progressbar.md)
-* [Formatting and Drawing](tutorial/formatting\_and\_drawing.md)
-* [Tagging](tutorial/tagging.md)
-* [Extensions](tutorial/extensions.md)
-* [Comparisons](tutorial/comparisons.md)
-* [Min/Max Operations](tutorial/minmax.md)
-* [Bitwise Operations](tutorial/bitwise.md)
 * [Table Lookups](tutorial/table\_lookups.md)
+
 * [Bit Extraction](tutorial/bit\_extraction.md)
-* [Truncating](tutorial/truncating.md)
 * [Rounding](tutorial/rounding.md)
+* [Truncating](tutorial/truncating.md)
 * [Floating Points](tutorial/floating\_points.md)
+
+* [Comparisons](tutorial/comparisons.md)
+* [Min/Max Operations](tutorial/minmax.md)
+* [Bitwise Operations](tutorial/bitwise.md)
+
+* [Common Tricks](tutorial/workarounds.md)
+* [Extensions](tutorial/extensions.md)
+
+* [Tagging](tutorial/tagging.md)
+
+## Compilation
+
+* [Composition](tutorial/composition.md)
+* [Reuse Arguments](howto/reuse\_arguments.md)
 * [Multi Precision](tutorial/multi\_precision.md)
 * [Multi Parameters](tutorial/multi\_parameters.md)
-* [Simulation](tutorial/simulation.md)
+* [Decorator](tutorial/decorator.md)
 * [Direct Circuits](tutorial/direct\_circuits.md)
 * [Statistics](tutorial/statistics.md)
```

Comment:
Statistics is more like formating & drawing !? Could be in analysis.

Reply:
It's something that we get directly by the compilation (it's not dynamic) so I thought it would make sense here, but we can move if it's the team preference